app/main.py

```diff
@@ -1 +1,41 @@
-#  write your code here
+class Animal:
+    def __init__(
+            self, name: str, appetite: int, is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> str:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        eaten = 0
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            eaten = self.appetite
+            self.is_hungry = False
+        return eaten
+
+
+class Cat(Animal):
+    def __init__(self, name: str, is_hungry: bool = True) -> None:
+        super().__init__(name, is_hungry=is_hungry, appetite=3)
```

Comment:
if u use named parameters I think it's better to pass name=name also, but I'm not confident)